python/tests/scripts/c3_pln_stv.metta

```diff
@@ -1,23 +1,33 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; The examples in this file is inspired by OpenCog Classic examples
+; from github.com/opencog/pln
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Some helpful definitions
 (= (min $a $b) (if (< $a $b) $a $b))
 (= (s-tv (stv $s $c)) $s)
 (= (c-tv (stv $s $c)) $c)
 
-; .tv serves as a special type of equality that requires
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ; `TV` "metarule" based on `match` to run
```

Comment:
Why description of .tv is dropped? I believe we need to add something about it, e.g., that it is a custom symbol, which is used to connect PLN-like expressions with their truth values. I'm not sure if we should go into detailed comparison with OpenCog Classic (e.g., that in this example .tv-expressions are ordinary expressions unlike OpenCog Classic, in which they are special type of values), or not. But some general comments on semantics of such custom symbols would be useful.

Reply:
I had moved the ".tv" description below, but I added a comment here to avoid confusion